docs/setup/kubernetes.md

```diff
@@ -1,5 +1,9 @@
 # Kubernetes
 
-To install use `helm install stable/hackmd`.
+HedgeDoc currently does not support any deployment via Kubernetes.
 
-For all further details, please check out the offical HedgeDoc  [K8s helm chart](https://github.com/kubernetes/charts/tree/master/stable/hackmd).
+If you want to help us creating a helm chart, then feel free to contact us on [Matrix](https://chat.hedgedoc.org/) or [Discourse](https://community.hedgedoc.org/).
+
+~~To install use `helm install stable/hackmd`.~~
```

Comment:
I would completely remove these lines.
Why should we first write "we don't support Kubernetes" and then mention a helm chart for a different project?

Reply:
I was not to sure if we want to remove it, so I opted to just strike through the old kubernetes documentation. But I'm more than willing to throw it out all together.

Reply:
In this case I would only remove the last line.

Reply:
I think this would yield more confusion than be helpful

Reply:
Why?.. `We don't support kubernetes. If you want to help us then contact us.` Why should that be confusing?

Reply:
the file has three lines and I guess you just saw the one I added and `
~~To install use 'helm install stable/hackmd'.~~`. But yes remove all the old documentation seems reasonable. Let me do that.